pkg/kv/kvserver/concurrency/lock_table_waiter.go

```diff
@@ -901,17 +924,65 @@ func (h *contentionEventHelper) emitAndInit(s waitingState) {
 			}
 			h.tBegin = timeutil.Now()
 		}
-	case waitElsewhere, waitQueueMaxLengthExceeded, doneWaiting:
+		h.sp.SetStatusTag(tagWaitKey, attribute.StringValue(s.key.String()))
```

Comment:
The analog to https://github.com/cockroachdb/cockroach/pull/71622 could also be helpful here. I know, you're trying to get us to a place where we have no need for such stop-gaps, but still, it might get us places for the foreseeable future. I would certainly appreciate having this in the debug.zip (where the labels [end up](https://github.com/cockroachdb/cockroach/pull/74070))